@ndlib/gatsby-theme-marble/src/components/Shared/ActionButtonGroup/DownloadButton/DownloadModalContent/DownloadCitation/test.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import DownloadCitation from './'
+
+describe('DownloadCitation', () => {
+  test('Unrendered', () => {
+    const manifest = {}
+    const wrapper = shallow(<DownloadCitation iiifManifest={manifest} />)
+    expect(wrapper.find('.citation').text()).toEqual('Unable to render citation')
+  })
+  test('Collection', () => {
+    const manifest = {
+      type: 'Collection',
+      slug: 'item/slug',
+      label: {
+        en: ['Title'],
+      },
+      provider: [{
+        homepage: [{
+          label: {
+            en: ['location'],
+          },
+        }],
+      }],
+    }
+    const wrapper = shallow(<DownloadCitation iiifManifest={manifest} />)
+    expect(wrapper.find('.citation').text()).toEqual('Title, slug. location, Hesburgh Libraries, University of Notre Dame, South Bend, IN. ' +
+      window.location.href)
+  })
+  test('Snite Manifest', () => {
+    const manifest = {
+      type: 'Manifest',
+      slug: 'item/slug',
+      label: {
+        en: ['Title'],
+      },
+      provider: [{
+        homepage: [{
+          label: {
+            en: ['Snite Museum of Art'],
+          },
+        }],
+      }],
+      metadata: [{
+        label:{
+          en:['Creator'],
+        },
+        value:{
+          en:['Creator'],
+        },
+      }, {
+        label:{
+          en:['Medium'],
+        },
+        value:{
+          en:['Medium'],
+        },
+      }, {
+        label:{
+          en:['Credit Line'],
+        },
+        value:{
+          en:['Credit Line'],
+        },
+      }, {
+        label:{
+          en:['Accession number'],
+        },
+        value:{
+          en:['Accession number'],
+        },
+      }],
+    }
+    const wrapper = shallow(<DownloadCitation iiifManifest={manifest} />)
+    expect(wrapper.find('.citation').text()).toEqual('Creator, Title, Medium. Snite Museum of Art, University of Notre Dame. Credit Line, Accession number.')
+  })
+  test('Snite Manifest', () => {
+    const manifest = {
+      type: 'Manifest',
+      slug: 'item/slug',
+      label: {
+        en: ['Title'],
+      },
+      provider: [{
+        homepage: [{
+          label: {
+            en: ['Rare Books and Special Collections'],
+          },
+        }],
+      }],
+      metadata: [{
+        label:{
+          en:['Creator'],
+        },
+        value:{
+          en:['Creator'],
+        },
+      }, {
+        label:{
+          en:['Date'],
+        },
+        value:{
+          en:['Date'],
+        },
+      }, {
+        label:{
+          en:['Collection'],
+        },
+        value:{
+          en:['Collection'],
+        },
+      }],
+    }
+    const wrapper = shallow(<DownloadCitation iiifManifest={manifest} />)
+    expect(wrapper.find('.citation').text()).toEqual('Creator. Title, Date. Collection. Rare Books and Special Collections, Hesburgh Libraries, University of Notre Dame, South Bend, IN.' +
+      window.location.href)
```

Comment:
Probably need to update test to not use `window`.

Reply:
Yeah, check unit tests. Also need to do a `mockImplementation` on the graphQl query now: ![Screen Shot 2020-06-12 at 2 47 40 PM](https://user-images.githubusercontent.com/416559/84536361-b1331c00-acbb-11ea-9dcf-26c9cb79ed04.png)